packages/regl-worldview/src/commands/GLText.js

```diff
@@ -354,7 +363,19 @@ function makeTextCommand(alphabet?: string[]) {
         let y = 0;
         let markerInstances = 0;
 
-        const fgColor = marker.colors?.[0] || marker.color || BG_COLOR_LIGHT;
+        const hitmapColor = (markerColor) => {
+          return {
+            r: markerColor?.[0] ?? 0,
+            g: markerColor?.[1] ?? 0,
+            b: markerColor?.[2] ?? 0,
+            a: markerColor?.[3] ?? 1,
+          };
+        };
```

Comment:
I'd lift this function definition to be standalone somewhere else in the file, since it'll get re-instantiated for every marker on every re-render.